elements/lisk-p2p/src/directory/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright © 2019 Lisk Foundation
```

Comment:
why `directory` instead of `peer_book`?

Reply:
Oh it was renamed back to `peer_directory`, `peer_book` is a more like a bucket manager that exposes the interface to handle peers whereas peer_directory covers the overall concept of peer lists and management